### ISSUE TITLE
get_library_upload_songs returning nothing

### ISSUE THREAD

**Issue by rderidder-lda** [sigma67/ytmusicapi#351]
Hello!
I am able to connect and create playlists, and they appear in yt under the right account, etc.
However when I call get_library_upload_songs, it returns nothing - though I have many songs.
This was working for me perfectly just less than a month ago.

I tried tracing it, and it seems like something around the fact that the SESSION_LIST has no 'contents', or, in the nav function it gets 2 items named contents, so the second one fails.. really not sure - but maybe that helps.

Basically, an exception is raised when it gets to the second 'contents' in the items parameter during the loop within the nav function (navigation.py).

I don't *think its my side - maybe something changed with yt's API?
else, let me know what it might be..
Thanks!

**Comment by sigma67** [comment]
It works just fine, make sure you've configured the correct account.

https://ytmusicapi.readthedocs.io/en/stable/usage.html#brand-accounts

**Comment by rderidder-lda** [comment]
weird..   what circumstances would allow it to create a playlist, but not get the uploads?
Also curious how it was working until I left it for awhile, and had to update the cookie.  Could it be something with the cookie? Again, not sure what is specific about the uploads vs the ability to create a playlist..
As mentioned.. I steppe through the code in debug.. and I see the response get tons of songs back.. but then the nav function returns nothing due to an exception with the second Contents item

**Comment by rderidder-lda** [comment]
response has lots of sings within in:
        response = self._send_request(endpoint, body)
     
but
        results = get_library_contents(response, MUSIC_SHELF)
returns nothing due to an exception within 
      contents = nav(response, SINGLE_COLUMN_TAB + SECTION_LIST, True)
   
it gets the exception when it hits the second Contents item (which is part of SECTION_LIST. The first Contents is in SINGLE_COLUMN_TAB)

**Comment by sigma67** [comment]
Again, do you have any brand accounts? Please check as instructed. https://myaccount.google.com/

And recreate your authentication credentials. Seeing that it has worked for you before it's unlikely to be a ytmusicapi issue.

**Comment by rderidder-lda** [comment]
trying it out.. same issue.. but now I don't think its seeing the songs like it was with the credentials above.. So original way has the songs in the response, but something doesn't quite work..   This Brand way doesn't show the songs, so maybe i need to play with it to get to the right account.   They have some strange account things going on it seems.

**Comment by rderidder-lda** [comment]
So from what i can understand, I do have a brand account - but this account is not the one with my music library.  If I enter ytmusic with it, i have to 'change accounts' under my profile image, and switch to the non brand account to see my library.

So that aligns with the above.  When I use credentials without the brand account #, it does get the songs in the response (I literally see the songs in the response object after the self._send_request call in uploads.py), but something seems to be malformed and the nav function throws an exception inside the results = get_library_contents(response, MUSIC_SHELF) call.

When I use the brand account #, it doesn't have any songs in the response.. but it also gets the exception at the second Contents item.

SO not sure if that gives any hints.. but from what i see, it connects correctly using the credentials.  Note that to further the point, I can create playlists with the yt object.. I just cant get_library_contents. 

Let me know if there is anything more I can provide.. Could it be something else with the headers_auth.json  contents that only effects getting library contents?

**Comment by rderidder-lda** [comment]
got it working.. was on version 0.24.1 of the ytmusicapi library.. I upgraded to 0.25.1 and its working now i think.. Still having other issues, but I think i'm past this one
